docs/migration.md

```diff
@@ -2526,9 +2526,9 @@ Windows Active Directory Source is not supported by the OpenTelemetry Collector.
 [telegraf-input-netstat]: https://github.com/SumoLogic/telegraf/tree/v1.24.3-sumo-4/plugins/inputs/netstat/README.md
 [telegraf-input-diskio]: https://github.com/SumoLogic/telegraf/tree/v1.24.3-sumo-4/plugins/inputs/diskio
 [telegraf-input-disk]: https://github.com/SumoLogic/telegraf/tree/v1.24.3-sumo-4/plugins/inputs/disk
-[dockerstatsreceiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.88.0/receiver/dockerstatsreceiver
-[dockerstatsmetrics]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.88.0/receiver/dockerstatsreceiver/documentation.md
+[dockerstatsreceiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.89.0/receiver/dockerstatsreceiver
+[dockerstatsmetrics]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.89.0/receiver/dockerstatsreceiver/documentation.md
 [sumologicschemaprocessor]: ../pkg/processor/sumologicschemaprocessor/README.md
```

Comment:
Are we going to change to the upstream version now or in the next release?

Reply:
I want to add the new `sumologic` processor and deprecate the `sumologic_schema` processor for this release, `v0.89.0-sumo-0`. I'm preparing a separate PR for this.

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/pull/1334